### DISCUSSION TITLE
logarithmic scalar bar

### DISCUSSION THREAD

**Discussion by Alex-Gauvain** [marcomusy/vedo#491] [Q&A] [answered]
Hello,

I would like to thank you for this great tool.
I would like to display the colour of my object with a logarithmic scale.
Can you help me?
Thanks in advance.

**Comment by marcomusy** [comment, marked as answer]
Thanks for your interest, at the moment genuine log scales are not supported, one way around could be to store/use the logarithm of the scalar and declare it as such in the scalarbar title. Let me know if need help on this.
